### DISCUSSION TITLE
Song % display in the Lazer editor

### DISCUSSION THREAD

**Discussion by Tenexxt** [ppy/osu#29683] [Ideas] [answered]
The editor in stable has a song % near the current time in the song.
The reason this should be added is the fact that the Ranking Criteria (or atleast the simplified ver. of it) says that you should use atleast ~80% of the song, and if you have a highly lower amount, you should cut the song or just map the rest.
The problem is the fact that Lazer doesn't have a song % display so you don't have a value that you can follow to follow that rule. You could technically guess the % but it should be obvious that you shouldn't follow rules on guessing if you can or can't do a thing.

**Comment by bdach** [comment, marked as answer]
There's a check for this implemented in the verify tab already.

**Reply by Tenexxt** [reply]
Should be enabled by default then, but thanks for answering
